client/src/modules/list/components/Items/model/actions.js

```diff
@@ -124,9 +124,7 @@ export const addItem = (item, listId) => dispatch =>
     .then(response => response.json())
     .then(json => {
       const action = addItemSuccess({ item: json, listId });
-      const { type, payload } = action;
 
-      socket.emit(type, payload);
       dispatch(action);
     })
```

Comment:
you can make it now shorter:
`json => dispatch(addItemSuccess({ item: json, listId })`